charts/incubator/docassemble/questions.yaml

```diff
@@ -0,0 +1,142 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{workload}
+# Include{workloadDeployment}
+
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+
+# Include{containerBasic}
+# Include{containerAdvanced}
+
+  - variable: docassemble
+    group: App Configuration
+    label: Docassemble
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: hostname
+          label: HostName
+          schema:
+            type: string
+            required: true
+            default: "http://localhost:10685"
+        - variable: https
+          label: https
+          schema:
+            type: boolean
+            default: false
```

Comment:
I think  this is only if you use the internal https

Reply:
also, wtf 4gb image? I dont think it will work on ci

Reply:
i never noticed the sized...its just that one tag too.